state/protocol/badger/state.go

```diff
@@ -910,32 +911,30 @@ func (state *State) populateCache() error {
 		if err != nil {
 			return fmt.Errorf("could not lookup finalized height: %w", err)
 		}
-		var cachedFinalHeader cachedHeader
-		err = operation.LookupBlockHeight(finalizedHeight, &cachedFinalHeader.id)(tx)
+		var cachedLatest cachedLatest
```

Comment:
My IDE complains: 
> Declaration of 'cachedLatest' shadows declaration at state.go

maybe we could 
1. update the name of this variable to:
   ```suggestion
		   var latest cachedLatest
   ```
2. move the variable declaration right after [line 907](https://github.com/onflow/flow-go/blob/357925d4b83cbe28b5d8c567de205cd3f5a6b230/state/protocol/badger/state.go#L907). The resulting code structure would be more intuitive for me: first declare the variable `latest`, then lookup and populate information about latest finalized block, then lookup and populate information about latest sealed block. At the moment, the variable declaration is just randomly somewhere in between. Not a problem ... but also not the most prettiest 😉  (sorry, my perfectionist tendencies coming through 😅 )